README.developer.md

```diff
@@ -377,9 +377,12 @@ The package build process can be adjusted using following environment variables:
 - `SKIP_BUILDDEP`
   - To skip installation of build dependencies this option should contain `yes` value.
 - `WITH_PYTHON`
-  - To skip building python bluechi modules this option should contain `0`.
+  - To skip building python bluechi modules this option should contain `0`. Default: `1`.
 - `WITH_COVERAGE`
-  - To start collecting coverage this option should contain `1`.
+  - To start collecting coverage this option should contain `1`. Default: `0`.
+- `WITH_SELINUX_PORT_ANY`
```

Comment:
Do we need that build option when both `bluechi_controller_port_bind_any` and `bluechi_agent_port_connect_any` are enabled by default? Do we really rebuild the package when we want to limit access to specific port only? Can't limitation to specific port be achieved only after installation by turning off those selinux booleans and run semanage?

Reply:
Good point! I removed it and verified that the default in the `bluechi.te` works as expected. Removed the `with_selinux_port_any` option from the spec file.